### ISSUE TITLE
Add a Language 🌎

### ISSUE THREAD

**Issue by jayehernandez** [jayehernandez/letra-extension#1]
Hey there!

Launched this extension with only five languages:
- 🇩🇪 German
- 🇫🇷 French
- 🇪🇸 Spanish
- 🇯🇵 Japanese
- 🇰🇷 Korean

I created a guide for adding a new language [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_language.md).

Here are some suggested languages you can add:
- 🇮🇳 Hindi
- 🇮🇹 Italian
- 🇷🇺 Russian

Comment here if you would like to work on adding a new language. Make sure that the voice is supported [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/supported_voices.md).

Please wait for a confirmation before starting on it! ✨

**Comment by Kvaibhav01** [comment]
Hey @jayehernandez, I would like to work on the Hindi translation.

**Comment by jayehernandez** [comment]
Hey @Kvaibhav01, thank you for your interest in contributing. Added a new issue for this [here](https://github.com/jayehernandez/letra-extension/issues/2).

Do leave a comment there so I can assign the issue to you! :)

**Comment by saurav-jaat** [comment]
hey @jayehernandez  I want to work on this issue.This is my first time contribution.Can you please help me out?I want to add English Language.

**Comment by Kingz2020** [comment]
Hi, I'd like to contribute to this project to. I am a first issuer.

**Comment by jayehernandez** [comment]
> hey @jayehernandez I want to work on this issue.This is my first time contribution.Can you please help me out?I want to add English Language.

Hey @saurav-jaat! Since the words are usually translated from one language to English right now, we can't support English as of the moment, sorry. Do you have any other language in mind? 

Alternatively, you can check out issue #13 which is an easy way to help out for first time contributors!

**Comment by jayehernandez** [comment]
> Hi, I'd like to contribute to this project to. I am a first issuer.

Hi @Kingz2020, what language do you want to add? Alternatively you can check out issue #13 which has the guide you need for a first time contributor!

**Comment by fluorspar20** [comment]
Hey @jayehernandez , I would like to work and contribute on the Russian translation!

**Comment by saurav-jaat** [comment]
Hey @jayehernandez,First of all thank you so much!! I have created my first PR with your help.Now,I want to add a french language.

**Comment by jayehernandez** [comment]
> Hey @jayehernandez , I would like to work and contribute on the Russian translation!

Hey @fluorspar20 thanks for showing interest in adding a new language, just want to ask if you speak Russian?

> Hey @jayehernandez,First of all thank you so much!! I have created my first PR with your help.Now,I want to add an Italian translation.

Saw your quote PR! Please wait for review on that. Just want to ask if you speak Italian?

**Comment by saurav-jaat** [comment]
@jayehernandez No i don't speak italian.

**Comment by jayehernandez** [comment]
> @jayehernandez No i don't speak italian.

@saurav-jaat sorry! Want to get people who speak the language so the words are accurate. Hope you understand.

If you want to contribute any other way, you can download the extension and suggest a feature you see fit!

**Comment by fluorspar20** [comment]
> 
> 
> > Hey @jayehernandez , I would like to work and contribute on the Russian translation!
> 
> Hey @fluorspar20 thanks for showing interest in adding a new language, just want to ask if you speak Russian?
> 
> > Hey @jayehernandez,First of all thank you so much!! I have created my first PR with your help.Now,I want to add an Italian translation.
> 
> Saw your quote PR, @saurav-jaat! Please wait for review on that. Just want to ask if you speak Italian?

No I don't speak Russian but I could help by translating each word into Russian using google translate if you are fine with it.

**Comment by jayehernandez** [comment]
> No I don't speak Russian but I could help by translating each word into Russian using google translate if you are fine with it.

@fluorspar20 sorry! Want to get people who speak the language so the words are accurate. Hope you understand.

If you want to contribute any other way, you can download the extension and suggest a feature you see fit!

**Comment by fluorspar20** [comment]
> 
> 
> > No I don't speak Russian but I could help by translating each word into Russian using google translate if you are fine with it.
> 
> @fluorspar20 sorry! Want to get people who speak the language so the words are accurate. Hope you understand.
> 
> If you want to contribute any other way, you can download the extension and suggest a feature you see fit!

Alright! Sounds fair. Thanks.

**Comment by chicoboymax** [comment]
Hi, thank you for this amazing extension. I would like to work on an Arabic translation.

**Comment by jayehernandez** [comment]
Hey @chicoboymax, thanks! Just want to ask if you speak Arabic? :)

**Comment by seryafarma** [comment]
Can I also contribute for Indonesian language? Indonesia is my mother tongue :indonesia:

**Comment by chicoboymax** [comment]
> 
> 
> Hey @chicoboymax, thanks! Just want to ask if you speak Arabic? :)

No I don't and just saw in other comments that you wanted people who speak the language !

**Comment by jayehernandez** [comment]
> > Hey @chicoboymax, thanks! Just want to ask if you speak Arabic? :)
> 
> No I don't and just saw in other comments that you wanted people who speak the language !

Thanks for understanding @chicoboymax! If you want, you can click [here](https://github.com/jayehernandez/letra-extension/projects/1) to work on some pending features.

**Comment by jayehernandez** [comment]
> Can I also contribute for Indonesian language? Indonesia is my mother tongue 🇮🇩

Hi @seryafarma, yes sure! Just put up a new issue for this #47, hope you can comment there so I can assign it to you. Thank you so much for contributing 🙏🏻

**Comment by seryafarma** [comment]
Sure! Picking it up right away. 💪

**Comment by TaniaMalhotra** [comment]
Hii!!
Can I work on the Hindi Female translation?

**Comment by jayehernandez** [comment]
> Hii!!
> Can I work on the Hindi Female translation?
> Hindi is my mother tongue

Hey @TaniaMalhotra!

Hindi is actually a supported language already. No need for a separate Female translation, however, you can revisit the translations [here](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/hindi.json) and check if you have any comments on it.

**Comment by jkfliu** [comment]
Hi @jayehernandez, can I add a Chinese language? I'm more familiar with the Simplified version (ZH-CN or ZH-HANS), if that's OK with you?

**Comment by jayehernandez** [comment]
Hi @jkfliu, sure! Good with using the Simplified version as well.

Created an issue for it here #113. Hope you can comment there so I can assign it to you.

**Comment by cassiocardoso** [comment]
Hi @jayehernandez, thanks for the amazing extension! Is there anyone working on adding Brazilian Portuguese? I would love to help with that (it's my native language 😄 ).

**Comment by parth298** [comment]
Thanks for the wonderful extension! You are doing well. I'm from India. I'm really New here. <a href="https://letsgetstartaffiliate.com/freelancing-work-market-presssoo/" rel="dofollow">#3</a>

**Comment by tobias1012** [comment]
I'd like to add Danish as a language

**Comment by Philippians-4-13** [comment]
I speak spanish and korean. 
Would that help? I'm new at this.

**Comment by Brummaha** [comment]
Hi @jayehernandez i speak Tamil, any good for you?!

**Comment by notorious-sobriety** [comment]
Hi @jayehernandez, do you still need Russian translation?  Russian is my native language.

**Comment by sakshi006** [comment]
> Hey there!
> 
> Launched this extension with only five languages:
> 
> * 🇩🇪 German
> * 🇫🇷 French
> * 🇪🇸 Spanish
> * 🇯🇵 Japanese
> * 🇰🇷 Korean
> 
> I created a guide for adding a new language [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_language.md).
> 
> Here are some suggested languages you can add:
> 
> * 🇮🇳 Hindi _(done by @Kvaibhav01! 💯)_
> * 🇮🇹 Italian
> * 🇷🇺 Russian
> 
> Comment here if you would like to work on adding a new language. Looking for people who **speak the language they want to add**, just so the words are accurate!
> 
> Make sure that the voice is supported [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/supported_voices.md).
> 
> Please wait for a confirmation before starting on it! ✨

Hey @jayehernandez, I would like to work on the Hindi translation. This is my first contribution.Can you please help me out?

**Comment by Kvaibhav01** [comment]
Hi @sakshi006, thanks for the interest but as you see the Hindi translation is already done. You can check out [other issues](https://github.com/jayehernandez/letra-extension/issues) to work on!

**Comment by n3twk** [comment]
Hi, I will work on Arabic, i'm a native Arab speaker.

**Comment by jayehernandez** [comment]
> Hi @jayehernandez, thanks for the amazing extension! Is there anyone working on adding Brazilian Portuguese? I would love to help with that (it's my native language 😄 ).

Hi @cassiocardoso, thank you! 🙏🏻 So sorry for the late revert, but Brazilian Portuguese was already added in #152. It wasn't documented here though. But you can check it out and see if the translations are correct!

**Comment by jayehernandez** [comment]
> I'd like to add Danish as a language

Hey @tobias1012, sure! Created a new issue for this #153, please comment there so I can assign it to you.

**Comment by jayehernandez** [comment]
> I speak spanish and korean.
> Would that help? I'm new at this.

Hi @Philippians-4-13, this is already supported but you can check the words in [spanish.json](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/spanish.json) and [korean.json](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/korean.json) and see if the translations are correct. You can also add **romanization** for Korean. Let me know if you would like to take this on, I can help you through it!

**Comment by jayehernandez** [comment]
> Hi @jayehernandez i speak Tamil, any good for you?!

Hi @Brummaha, yes! You can work on the Tamil translation. Created an issue for this #154, please comment there so I can assign it to you.

> Hi @jayehernandez, do you still need Russian translation? Russian is my native language.

Hi @notorious-sobriety, it's not yet supported so you can take this on! Created an issue for this #155, please comment there so I can assign it to you. :) 

> Hi, I will work on Arabic, i'm a native Arabic speaker.

Hi @n3twk, sure! #156 is the issue I created for this, please comment there so I can assign it to you.

**Comment by Philippians-4-13** [comment]
> > I speak spanish and korean.
> > Would that help? I'm new at this.
> 
> Hi @Philippians-4-13, this is already supported but you can check the words in [spanish.json](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/spanish.json) and [korean.json](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/korean.json) and see if the translations are correct. You can also add **romanization** for Korean. Let me know if you would like to take this on, I can help you through it!

I can help you with the romanization for Korean!
I'll try my best but it would be nice if you can help me thou. I'm a newbie in Github.

**Comment by jayehernandez** [comment]
> korean.
> Would that help? I'm new at this.

Opened a new issue for you #157. Please comment there so I can assign it to you. We can also talk there about how we’ll go through this.

**Comment by jayehernandez** [comment]
> > > I speak spanish and korean.
> > > Would that help? I'm new at this.
> > 
> > 
> > Hi @Philippians-4-13, this is already supported but you can check the words in [spanish.json](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/spanish.json) and [korean.json](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/korean.json) and see if the translations are correct. You can also add **romanization** for Korean. Let me know if you would like to take this on, I can help you through it!
> 
> Hey, @jayehernandez I can help you with the romanization for Korean!
> I'll try my best but it would be nice if you can help me thou. I'm a newbie in Github.

@Philippians-4-13 Opened a new issue for you #157. Please comment there so I can assign it to you. We can also talk there about how we’ll go through this.

**Comment by userakai** [comment]
Hi, I could help with the Italian version! I'm very new to json files but I think I could be of use.

Also, if I may suggest a small change: Japanese kanji are hard to read without the phonetic characters above (Furigana, as in the picture below), so I can't understand what I'm reading without searching on the dictionary. If you could implement a way to add furigana it would be awesome for us Japanese learners.

![furigana](https://user-images.githubusercontent.com/65029150/84794162-35580d00-aff6-11ea-911c-13da5ddd8432.png)

**Comment by T3yk1L** [comment]
Hey, I could help with Russian language, I'm new to Git and GitHub but I'd really like to help.

**Comment by shamsrza** [comment]
Hi, I can help with Turkish and Russian female. I am new in Github as well and therefore I need some support and direction. Also I suggest to add Azeri language (my native language).

**Comment by Alireza-Shie** [comment]
Hi, I want to work on Persian

**Comment by Prenda77** [comment]
> Hey @Kvaibhav01, thank you for your interest in contributing. Added a new issue for this [here](https://github.com/jayehernandez/letra-extension/issues/2).
> 
> Do leave a comment there so I can assign the issue to you! :)
Well, I would like to see & ready something, in Portuguese, here... 🤗

**Comment by sanzhar999** [comment]
Hey, @jayehernandez. I want to help you with Russian translation and yes, I speak Russian.

**Comment by Kvaibhav01** [comment]
Hey @sanzhar999, @notorious-sobriety is assigned on the Russian translation currently. #155

**Comment by sanzhar999** [comment]
@Kvaibhav01 ooo, OK. I dont know. But, if u need my help. I am always ready

**Comment by jayehernandez** [comment]
> Hi, I could help with the Italian version! I'm very new to json files but I think I could be of use.
> 
> Also, if I may suggest a small change: Japanese kanji are hard to read without the phonetic characters above (Furigana, as in the picture below), so I can't understand what I'm reading without searching on the dictionary. If you could implement a way to add furigana it would be awesome for us Japanese learners.
> 
> ![furigana](https://user-images.githubusercontent.com/65029150/84794162-35580d00-aff6-11ea-911c-13da5ddd8432.png)

Hi @userakai, created an issue for adding Italian in #185, please comment there so I can assign it to you. :)

Great suggestion on adding Furigana as well, I’m not learning Japanese myself but we have support for adding romanization. Would that be more helpful? Will just need help on filling in the data for that though.

**Comment by jayehernandez** [comment]
> Hey, I could help with Russian language, I'm new to Git and GitHub but I'd really like to help.

Hey @T3yk1L, Russian is already supported as done in issue #155. If you want to practice contributing to open source, you can go to issue #13, this is great for beginners!

**Comment by jayehernandez** [comment]
> Hi, I can help with Turkish and Russian female. I am new in Github as well and therefore I need some support and direction. Also I suggest to add Azeri language (my native language).

Hey there @shamsrza, Russian is already supported as stated in issue #155.

I created a new issue for adding Turkish here #186. Please comment there so I can assign it to you. You can follow the instructions written there and if you need help, just comment there as well.

Sadly, responsive voice (the software used for text to speech) does not support Azeri so we can’t add it yet.

**Comment by userakai** [comment]
> Great suggestion on adding Furigana as well, I’m not learning Japanese myself but we have support for adding romanization. Would that be more helpful? Will just need help on filling in the data for that though.

Romanization is the next best thing, it makes it much more helpful. I'm sorry I can't help you with that since I'm just learning japanese and could make some mistakes.

**Comment by jayehernandez** [comment]
> Hi, I want to work on Persian

Hey there @Alireza-Shie, Persian is sadly not supported by Responsive Voice, sorry! Hope you can take a look at the other issues if you would like to contribute. :)

**Comment by jayehernandez** [comment]
> > Great suggestion on adding Furigana as well, I’m not learning Japanese myself but we have support for adding romanization. Would that be more helpful? Will just need help on filling in the data for that though.
> 
> Romanization is the next best thing, it makes it much more helpful. I'm sorry I can't help you with that since I'm just learning japanese and could make some mistakes.

No worries @userakai, will open it up to other people who can help out :)

**Comment by dogagcmnr** [comment]
Dear, I'm interested in making Turkish translation for your project. Please let me know.. Best wishes..

**Comment by Puja304** [comment]
Hey! Can I help out with the Hindi translation? I am new to GitHub so I really don't know how to get started

**Comment by jayehernandez** [comment]
> Dear, I'm interested in making Turkish translation for your project. Please let me know.. Best wishes..

Assigning Turkish to you as you seem to already have opened up a PR #196. 

@shamsrza sorry someone beat you to the Turkish translation!

**Comment by jayehernandez** [comment]
> Hey! Can I help out with the Hindi translation? I am new to GitHub so I really don't know how to get started

Hey there, Hindi is already supported. This was done by @Kvaibhav01. If you want to contribute, you can check out issue #13. This is beginner friendly!

**Comment by naman2807** [comment]
Hi, I'd like to contribute to this project to.This is my first priject

**Comment by HakaCode** [comment]
I am new to contributing and I would like to contribute in 🇧🇷 Brazilian Portuguese

**Comment by jayehernandez** [comment]
> I am new to contributing and I would like to contribute in 🇧🇷 Brazilian Portuguese

Hey @HakaCode we already support this here! sorry!

**Comment by jayehernandez** [comment]
> Hi, I'd like to contribute to this project to.This is my first priject

Hello @naman2807, please checkout #13, this is good for beginners!

**Comment by IC1101Virgo** [comment]
Hi ! I can work on German Translation..

**Comment by Kvaibhav01** [comment]
Hi @IC1101Virgo, thanks for the participation but [German is already supported](https://github.com/jayehernandez/letra-extension#-supported-languages). You are welcome to work on some [existing (un-assigned) issues](https://github.com/jayehernandez/letra-extension/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee).

**Comment by IC1101Virgo** [comment]
@Kvaibhav01 Thank you so much. I'll look them up 😄

**Comment by Wanrick** [comment]
Hi 👋 
I am new to contributing. I would like to contribute Afrikaans as a language option.
(I accidentally created the issue before coming here. Apologies.)

**Comment by jayehernandez** [comment]
Hi @Wanrick, thanks for the interest in contributing! No worries about the accidental PR. Created a new one though #247 please comment there so I can assign it to you.

Thank you!

**Comment by meghsohor** [comment]
Thank you @jayehernandez for accepting my commit of #255 . It was my first contribution to an open-source project. I am really happy that I could work with my native language and you have given me that opportunity. Hopefully I will be able to make more contribution to this project.

Cheers!

**Comment by jayehernandez** [comment]
Congrats on your first PR @meghsohor! Hoping it’s the first of many. The project’s always open for more people to contribute in any way they can. :)

**Comment by elaynelemos** [comment]
Hey! May I work on adding PT-BR?

**Comment by HakaCode** [comment]
> Ei! Posso trabalhar na adição de PT-BR?

@elaynelemos https://github.com/jayehernandez/letra-extension/issues/1#issuecomment-658497425

**Comment by jayehernandez** [comment]
Thanks @HakaCode! @elaynelemos, since this is already supported, if you find that there's better translations for the words, feel free to submit a PR to discuss!

**Comment by jebitok-dev** [comment]
I would like to add Kiswahili as a language.

**Comment by mViolet** [comment]
Hi - I would love to add Croatian as a language!

**Comment by itzmeowww** [comment]
I would like to add the Thai language. (I've already added 😅)

**Comment by mViolet** [comment]
Hey! I would love to add Croatian as a language.

**Comment by yuxxeun** [comment]
Hey @jayehernandez can I contribute to Indonesian?

**Comment by Kvaibhav01** [comment]
Hey @yuxxeun, the Indonesian language is [already supported](https://github.com/jayehernandez/letra-extension/pull/65)!

**Comment by yuxxeun** [comment]
> Hey @yuxxeun, the Indonesian language is [already supported](https://github.com/jayehernandez/letra-extension/pull/65)!

yes, thank you for the information, actually I have read on the readme that Indonesian already exists, but I offer to be ready to know something is missing. I really want to contribute but confused about what contribution :((

**Comment by Kvaibhav01** [comment]
Hey, @yuxxeun look at the [different issues](https://github.com/jayehernandez/letra-extension/issues) that are opened and aren't claimed by anyone yet. You can work on those. Also, if you try to run the project in your local machine, you might see some bugs or issues which you can file and work. Maybe you'll get some new ideas/features!

**Comment by elenakout** [comment]
Can I contribute Greek language?

**Comment by xanaDev** [comment]
Can I add Arabic Language ? (I can speak the language)

**Comment by markrofail** [comment]
@jayehernandez would like to contribute to the Arabic Language. Native Arabic speaker.

**Comment by divn** [comment]
I could work on Finnish if its ok

**Comment by Kvaibhav01** [comment]
@markrofail Arabic is already supported. Please check out the [Readme](https://github.com/jayehernandez/letra-extension/blob/master/README.md) and the _[words](https://github.com/jayehernandez/letra-extension/tree/master/server/data/words)_ folder to see which languages are already supported.

**Comment by DevelopMan** [comment]
@jayehernandez I would like to contribute on the Russian language. I am a native Russian. :)

**Comment by Kvaibhav01** [comment]
@DevelopMan Russian is [already supported](https://github.com/jayehernandez/letra-extension/blob/master/server/data/words/russian.json). Please check out the [Readme](https://github.com/jayehernandez/letra-extension/blob/master/README.md) and the _[words](https://github.com/jayehernandez/letra-extension/tree/master/server/data/words)_ folder to see which languages are already supported.

**Comment by DevelopMan** [comment]
@Kvaibhav01  Sorry, I didn't check the file, but the best option will be to change the issue text. ;)
![add_lang_screenshot](https://user-images.githubusercontent.com/2499467/95310418-aadd3700-088c-11eb-9340-1ed9c91ebd37.png)

**Comment by Kvaibhav01** [comment]
For sure @DevelopMan, I'll let Jaye know about this. Thanks!

**Comment by adico1** [comment]
Can contribute Hebrew, 
but I think voice is not supported

**Comment by Kvaibhav01** [comment]
Hi @adico1, yes. As per the ResponsiveVoice API voices Hebrew isn't supported as listed [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/supported_voices.md).

**Comment by jayehernandez** [comment]
Hey @DevelopMan thanks for letting me know about this. Updated the description to redirect other contributors to the README for the full list. Thanks @Kvaibhav01 for letting me know about this as well.

**Comment by jayehernandez** [comment]
Hey @mViolet, created an issue for adding Croatian in #418. Please comment there so I can assign it to you.

Hey @elenakout, created an issue for adding Greek in #417. Please comment there so I can assign it to you.

**Comment by WyldWanderer** [comment]
@jayehernandez Sadly I can't contribute another language but I did write a script that auto-fills the Google sheet's 'word' column using the Google Translate formula you provided. It would cut down on time needed as someone could fill in all the words from Google Translate and then review the words from there. Before I submit a pull request however I wanted to check with you to see if such a tool is in line with your goals for this project.

**Comment by jayehernandez** [comment]
Hey @WyldWanderer yes this is great! Please go ahead with opening up the PR. We can also talk about the needed documentation changes when we go with that route.

**Comment by WyldWanderer** [comment]
> Hey @WyldWanderer yes this is great! Please go ahead with opening up the PR. We can also talk about the needed documentation changes when we go with that route.

Sounds good, PR is open!

**Comment by AugustasV** [comment]
I would like to add Lithuanian translation!

**Comment by jzhengcode** [comment]
@jayehernandez Hello! I see that there is support for Simplified Chinese and I would love to contribute Traditional Chinese

**Comment by jayehernandez** [comment]
Hi @jzhengcode sorry, this is already in progress by PR #470 😅

**Comment by DavidtKate** [comment]
Hello @jayehernandez. I could work on a Dutch translation as this is my native language and it has male/female supported voice.

**Comment by Dave-mash** [comment]
> Hey there!
> 
> I created a guide for adding a new language [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/new_language.md).
> 
> All currently supported languages are stated in the [README](https://github.com/jayehernandez/letra-extension/blob/master/README.md#-supported-languages).
> 
> Comment here if you would like to work on adding a new language. Looking for people who **speak the language they want to add**, just so the words are accurate!
> 
> Make sure that the voice is supported [here](https://github.com/jayehernandez/letra-extension/blob/master/docs/supported_voices.md).
> 
> Please wait for a confirmation before starting on it!

Hey @jayehernandez, I would like to work on the Swahili translation

**Comment by Dave-mash** [comment]
Hey @jayehernandez, I would like to work on the Swahili translation

**Comment by Kvaibhav01** [comment]
@Dave-mash Swahili language work is already assigned. #359

**Comment by jayehernandez** [comment]
@DavidtKate saw your PR, thank you!

**Comment by shehriyarnadeem** [comment]
I would like to work on Urdu language. But it has no supported voice. Is it possible to use hindi voice for it due to similarity ?

**Comment by jayehernandez** [comment]
@shehriyarnadeem, sorry since it doesn't have a supported voice, we will not be able to add it for now. Using a similar voice may cause inaccuracy if people who want to learn the language use it.

**Comment by juppsy** [comment]
Hi @jayehernandez! I would like to translate for Romanian language.

**Comment by yunsuklee** [comment]
Hi @jayehernandez! May I contribute adding a Spanish translation? I am a native Spanish speaker, born and raised in Argentina although it may not seem like it

**Comment by sharvarii11** [comment]
hey, i would like to contribute on hindi translation , can I?

**Comment by inarie** [comment]
Hi @jayehernandez, can I contribute with portuguese 🇵🇹 ?

**Comment by Kvaibhav01** [comment]
Hi @sharvarii11, the Hindi language translation is already done, as you can see in the readme's [supported languages list](https://github.com/jayehernandez/letra-extension#-supported-languages).

@inarie, as per the [latest release notes](https://github.com/jayehernandez/letra-extension/releases/tag/v1.0.15):

> No new issues and pull requests will be entertained. The repository will be revamped to reflect this change.

**Comment by AkaraTawng** [comment]
@jayehernandez I'd like to work on the Lao translation if possible (:
